extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php

```diff
@@ -122,7 +122,10 @@ public function track($amgId, $gracenoteId, Title $title) {
 
 		wfDebug(__METHOD__ . ': ' . json_encode($data) . "\n");
 
-		$resp = Http::post($url, ['postData' => $data]);
+		$resp = Http::post($url, [
```

Comment:
I wonder about creating something like `ExternalHttp::post` which would also help us when we get to implementing HTTP headers for request tracing, what do you think? or maybe leave `Http` as it is and introduce `InternalHttp` ?